BAKKA_Editor/Views/MainView.axaml

```diff
@@ -131,6 +131,16 @@
                 <MenuItem Header="{DynamicResource L.InitialChartSettingsMenuOption}"
                           Command="{Binding OpenInitialChartSettings}" />
             </MenuItem>
+			<MenuItem Header="_Options">
```

Comment:
We shouldn't be adding any more new things here since it's getting way too long. Ideally, this should be in the settings. For now I'm okay with just adding another checkbox in the settings UI since we don't have a proper UI yet.